packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/ola-step-checker/ola-step-checker.controller.js

```diff
@@ -49,4 +31,11 @@ export default class {
       maximumSignificantDigits: 1,
     }).format(price);
   }
+
+  hasBandwidthExtension() {
+    const specs = filter(this.ola.availableModes, {
+      interfaces: [{ count: 4, aggregation: true }],
+    });
+    return specs.length > 0;
```

Comment:
for check if [] have value inside, isn't optimal do `!!specs` to transform on a boolean ? however, if is good, can directly return !!filter() ^^

Reply:
what do you suggest please ? i don't understand your comment :)
```
!![3] === true // true
```